fastly/observability_custom_dashboard.go

```diff
@@ -0,0 +1,275 @@
+package fastly
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+const path = "/observability/dashboards"
```

Comment:
For consistency, I'd prefer to drop this and just use `ToSafeURL` everywhere. At some point in the future we'll probably improve ToSafeURL to accept a mixture of slices and strings, at which point this could be a 'const slice of strings' and passed as the first argument to ToSafeURL, but that day is not today.